syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.datashare";
option java_outer_classname = "DataShareProto";
option objc_class_prefix = "DS";

package datashare;

service DataShare {
  rpc SendData (ClientRequest) returns (ServerReply) {}
}

message ClientRequest {
  string filename = 1;
}
message ServerReply {
  int32 number = 1;
  string message = 2;
  bytes file = 3;
}
